beacon_node/http_api/src/validators.rs

```diff
@@ -18,6 +18,8 @@ pub fn get_beacon_state_validators<T: BeaconChainTypes>(
             |state, execution_optimistic, finalized| {
                 let epoch = state.current_epoch();
                 let far_future_epoch = chain.spec.far_future_epoch;
+                let ids_filter_set: Option<HashSet<&ValidatorId>> =
+                    query_ids.as_ref().map(|f| HashSet::from_iter(f));
```

Comment:
Clippy wants to simplify this closure:

```suggestion
                    query_ids.as_ref().map(HashSet::from_iter);
```